frontend/src/plugins/veeValidate/__tests__/oneEmojiOrTwoCharacters.spec.js

```diff
@@ -0,0 +1,29 @@
+import oneEmojiOrTwoCharacters from '../oneEmojiOrTwoCharacters.js'
+
+const mockI18n = {
+  $tc: (key) => key,
+}
+
+describe('oneEmojiOrTwoCharacters validation', () => {
+  it.each([
+    ['1', true],
+    ['12', true],
+    ['123', false],
+    ['🧑🏼‍🔧', true],
```

Comment:
Does one of these tests check for combining emojis? Such as skin color combiners etc.. I'm not an expert on this, so I don't know whether one of your tests already accounts for that.

Reply:
This is already one that is made out of 3 combined emojis.

> The Man Mechanic: Light Skin Tone emoji is a [ZWJ sequence](https://emojipedia.org/emoji-zwj-sequence) combining [👨 Man](https://emojipedia.org/man), [🏻 Light Skin Tone](https://emojipedia.org/light-skin-tone), [Zero Width Joiner](https://emojipedia.org/zero-width-joiner) and [🔧 Wrench](https://emojipedia.org/wrench).
– Emojipedia

Reply:
That's also the reason, that I've included some basic smileys below that should be each counted as one.